samplers/samplers.go

```diff
@@ -80,6 +101,22 @@ type JSONMetric struct {
 	Value []byte `json:"value"`
 }
 
+const sinkPrefix string = "veneursinkonly:"
+
+func routeInfo(tags []string) RouteInformation {
+	var info RouteInformation
+	for _, tag := range tags {
+		if !strings.HasPrefix(tag, sinkPrefix) {
+			continue
+		}
+		if info == nil {
+			info = make(RouteInformation)
+		}
+		info[tag[len(sinkPrefix):]] = struct{}{}
```

Comment:
As discussed in chat, this was a bit confusing. Maybe a comment for "if we find an appropriate tag we'll make entry in the routing map with the sink name as the key and an a useless key, since we really just want to check for the key later" or something less wordy. :)